dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/testHandlerProvider/src/com/ibm/samples/jaxws/testhandlerprovider/SayHelloServiceWithHandlerImpl.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.samples.jaxws.testhandlerprovider;
+
+import javax.jws.HandlerChain;
+import javax.jws.WebService;
+
+import com.ibm.samples.jaxws.testhandlerprovider.service.SayHelloService;
+
```

Comment:
Can add code comments to explain what this Web Service is used for, like: 

```suggestion
// Web Service used to test optional elements added to responses. 
```